front/src/applications/operationalStudies/hooks/useManageTrainScheduleContext.tsx

```diff
@@ -16,6 +22,8 @@ type ManageTrainScheduleContextType = {
   launchPathfinding: (pathSteps: (PathStep | null)[]) => void;
   pathfindingState: PathfindingState;
   infraInfo: { infra?: InfraWithState; reloadCount: number };
+  /** Operational points along the path (including origin and destination) and vias added by clicking on map */
+  allWaypoints?: SuggestedOP[];
```

Comment:
The original comment is bogus: the list also contains all suggested OPs. The name `allWaypoints` is also confusing for the same reason.

Maybe we should rename this to something else while at it? For instance, `waypointsAndSuggestedOPs`?